Branch-SDK/Branch-SDK/BranchUniversalObject.h

```diff
@@ -12,6 +12,8 @@
 #import <UIKit/UIKit.h>
 #endif
 
+#import "Branch.h"
+#import <CoreSpotlight/CoreSpotlight.h>
```

Comment:
This may need to do what Ed did elsewhere recently, something like:

```Obj-C
#if __has_feature(modules)
@import CoreSpotlight;
#else
#import <CoreSpotlight/CoreSpotlight>
#endif
```

Might want to verify with @E-B-Smith.

Reply:
Yep -- I'm adding these now.